src/libs/actions/User.ts

```diff
@@ -1385,6 +1385,13 @@ function setIsDebugModeEnabled(isDebugModeEnabled: boolean) {
     Onyx.merge(ONYXKEYS.USER, {isDebugModeEnabled});
 }
 
+/**
+ * Check which policies the user can join
+ */
+function validateUserAndGetAccessiblePolicies(validateCode: string) {
+    Session.validateUserAndGetAccessiblePolicies(validateCode);
+}
```

Comment:
This method looks pointless. Just call `Session.validateUserAndGetAccessiblePolicies` directly.

Reply:
Handled.